velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2655,6 +2556,13 @@ TEST_F(DateTimeFunctionsTest, dateDiffTimestampWithTimezone) {
 }
 
 TEST_F(DateTimeFunctionsTest, parseDatetime) {
+  auto parseDatetime = [&](const std::optional<std::string>& input,
+                           const std::optional<std::string>& format) {
+    auto timestampWithTimezone = evaluateOnce<TimestampWithTimezone>(
```

Comment:
@svm1 with this PR you can start using evaluateOnce() to return TimestampWithTimezone

Reply:
Thanks, this definitely removes a lot of the overhead and simplifies testing with TimestampWithTimezone type.